converter/internal/common/common.go

```diff
@@ -49,14 +50,23 @@ func getValueOverrideHook() builder.ValueOverrideHook {
 	}
 }
 
-// GetUniqueLabel appends a counter to the input label if it is not the first.
-// The input label is 1-indexed.
-func GetUniqueLabel(label string, currentCount int) string {
-	if currentCount == 1 {
-		return label
+func GetLabelForParts(parts ...interface{}) string {
+	var sParts []string
+	for _, part := range parts {
+		if part != "" {
+			sParts = append(sParts, fmt.Sprintf("%v", part))
+		}
+	}
+	return strings.Join(sParts, "_")
+}
+
+func GetLabelWithIndex(index int, parts ...interface{}) string {
```

Comment:
Two nits:

1. It's slightly unidiomatic for functions to get Get in the name; these would normally be `LabelForParts` and `LabelWithIndex`
2. These should have some Godoc comments explaining to people writing converters what they do and how to use them :)